jmxtrans-output/jmxtrans-output-core/src/main/java/com/googlecode/jmxtrans/model/output/support/opentsdb/OpenTSDBMessageFormatter.java

```diff
@@ -64,6 +66,7 @@
 
 	private final boolean mergeTypeNamesTags;
 	private final String hostnameTag;
+	private Server server;
```

Comment:
Server should not be part of the state of the message formater. The same message formater can be used to format messages from multiple different servers, and can do that concurrently.